lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -287,15 +287,49 @@ def get_stack_events(cfn, stack_name):
         return ret
 
     for e in events.get('StackEvents', []):
-        eventline = 'StackEvent {ResourceType} {LogicalResourceId} {ResourceStatus}'.format(**e)
-        ret['events'].append(eventline)
-
-        if e['ResourceStatus'].endswith('FAILED'):
-            failline = '{ResourceType} {LogicalResourceId} {ResourceStatus}: {ResourceStatusReason}'.format(**e)
-            ret['log'].append(failline)
+        if e['Timestamp'] > started_at:
```

Comment:
Perhaps instead of Timestamp, we could filter events based on the last `_COMPLETE` event affecting the top-level `AWS::CloudFormation::Stack`. That would get us around any timezone funny business. 

I think we'd want to include everything after a `UPDATE_COMPLETE`, `UPDATE_ROLLBACK_COMPLETE`, or `CREATE_COMPLETE`.